src/web-extension/manifest.json

```diff
@@ -34,5 +34,15 @@
     "default_icon": "icon-128.png",
     "default_popup": "popup.html"
   },
+  "commands": {
+    "_execute_browser_action": {
+      "suggested_key": {
+        "default": "Ctrl+T",
+        "mac": "MacCtrl+T",
+        "chromeos": "Ctrl+T",
+        "linux": "Ctrl+T"
```

Comment:
Do we actually need `"chromeos"` and `"linux"` because they're the same as the default?

At least according to the [docs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/commands):
> The value for `"default"` is used on all platforms that are not explicitly listed.

Reply:
A note in our `README.md` would be nice indeed.
Otherwise users might never find out about this feature.

Reply:
👍 on both. Was more of a quick hack to try it out.